hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/CompleteMultipartUploadRequest.java

```diff
@@ -51,10 +51,10 @@ public void setPartList(List<Part> partList) {
   @XmlRootElement(name = "Part")
   public static class Part {
 
-    @XmlElement(name = "PartNumber")
+    @XmlElement(name = "PartNumber", namespace = OzoneConsts.S3_NAMESPACE_URL)
```

Comment:
Could you help check whether we can simply set the namespace to the enclosing class's `@XmlRootElement` (i.e `@XmlRootElement(name = "Part", namespace = ""http://s3.amazonaws.com/doc/2006-03-01/")?

If so, we only need to modify the `@XmlRootElement` annotations and there is no need to change the `@XmlElement` annotations.